src/content/en/2022/seo.md

```diff
@@ -1,22 +1,755 @@
+
+{{ figure_markup(
+  image="anchor-rel-attr.png",
+  caption="29.5% of links with rel attributes implement nofollow.",
+  description="A column chart comparing rel noopener, nofollow, noreferrer, dofollow, sponsored, ugc, and follow.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSZxId5PPpTHp4835BI-rt4HqZVrhn1UiF0xcOrVUytVs6j262ZqFrY5e1FXZibAGDY6gWnZ0uG3WZC/pubchart?oid=583878647&format=interactive",
+  sheets_gid="411053146",
+  sql_file="anchor-rel-attribute-usage.sql"
+  )
+}}
+
+
+Not much has changed in terms of the adoption of the newer attributes, with `rel=ugc` appearing on 0.4% of desktop and mobile pages, and `rel=sponsored` appearing on 0.5% of desktop and 0.4% of mobile pages in 2022.
+
+`rel="dofollow"` once again appeared on more pages than `rel="ugc"` and `rel="sponsored"`. While this is technically not a problem, Google ignores `rel="follow"` and `rel="dofollow"` because, despite their inclusion, they are not actually official attributes.
+
+`rel="nofollow"`, which is a real attribute, was found in 2022 on 29.5% of mobile pages, which is 1.2% less than last year. Google treats `nofollow` as a hint, meaning the search engine can choose whether or not they respect the attribute.
+
+
+## AMP
+
+AMP has been a controversial topic since its launch in 2015, with SEOs debating whether or not it had a direct impact on rankings. Google later released this statement (below) in its documentation for additional clarification:
+
+<figure>
+  <blockquote>While AMP itself isn’t a ranking factor, speed is a ranking factor for Google Search. Google Search applies the same standard to all pages, regardless of the technology used to build the page.</blockquote>
+  <figcaption>— <cite><a hreflang="en" href="https://developers.google.com/search/docs/advanced/experience/about-amp">Google Search Central</a></cite></figcaption>
+</figure>
+
+The future of AMP appears to be changing ever since the launch of Core Web Vitals. A main reason for previously implementing AMP, aside from improving page speed, was that it was necessary for inclusion in Top Carousels. In 2021, Google updated its requirements and outlined that any page is now eligible to appear in Top Carousels with or without AMP.
+
+{{ figure_markup(
+  image="amp-markup.png",
+  caption="Amp adoption is low across the sampled pageset.",
+  description="A column chart showing AMP implementations across all pages.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSZxId5PPpTHp4835BI-rt4HqZVrhn1UiF0xcOrVUytVs6j262ZqFrY5e1FXZibAGDY6gWnZ0uG3WZC/pubchart?oid=1100298070&format=interactive",
+  sheets_gid="1210998028",
+  sql_file="markup-stats.sql"
+  )
+}}
+
+Desktop usage has dipped in 2022 from 0.09% to 0.07% compared to 2021 while mobile usage is down from 0.22% to 0.19% over the same time period.
+
+## Internationalization
+
+### Hreflang usage
```

Comment:
Again can we add an intro text between these headings?

Reply:
Can we add: 

Internationalization in SEO is the process of optimising a website in line with best practices when targeting multiple countries and multiple languages, to ensure that it can be properly crawled and indexed by search engines.

Reply:
Added.